Update index.html

CriticalWasTaken opened the pull request:
Began the foundation of the website.

CriticalWasTaken committed:
Update index.html

Began the foundation of the website.

dubinsky commented on index.html:
```diff
@@ -1,12 +1,70 @@
+<html>
 
+<head></head>
+<style>
```
Please move the styling into a dedicated .css file and associate it with the HTML in the `<head>`.

CriticalWasTaken replied:
I actually don't know how to tell the html to read a separate css file. Would you mind showing me tomorrow (I think we have a meeting tomorrow morning)?

dubinsky replied:
I am unlikely to be available  tomorrow morning.
see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
You need to add a `<link>` element with attribute `rel="stylesheet"` to the `<head>` of the page.

CriticalWasTaken replied:
Ah ok. So I make a separate CSS file and tell the HTML to access it using <link> in the <head>. Fine, I'll implement this, and make a pull request for the CSS file as well.

dubinsky replied:
Right. It is better to include the `.css` file in the same pull request though, since the changes are related. I think there is a way to add to the existing pull request using GitHub's web interface, and this is certainly possible using git from the command line (or via a proper integrated development environment with git integration).

CriticalWasTaken replied:
Ok, so would you like me to close this pull request, make the changes (new css file and footer element) and then make a new pull request?

dubinsky replied:
No, I would like you to package all the changes, including the `.css` file in this pull request - if you figure out how.

CriticalWasTaken replied:
I don't think it is possible in the current form. What I can do, is make the css file in a separate request, then edit this request to use a <link> to access it. Shall I do that? (I'm not sure via github how to add a new file to the current pull. I can simply make a new one and then have this html be able to access it.)

dubinsky replied:
sure, if you can't do the change in one pull request, split it.
I think the way to modify an existing pull request (e.g., add the `.css` file to this one) is to edit the code on the branch corresponding to the pull request (in this case,https://github.com/mathworlds/site/tree/CriticalWasTaken-patch-4). You can see the list of all branches at https://github.com/mathworlds/site/branches
By the way, you may want to delete the branches that are no longer relevant.

CriticalWasTaken replied:
I'm not sure which branches are supposed to be there and which are defunct, as I didn't add any of them. So I haven't touched them. I'll add the css pull request in a moment, and then edit this pull.

dubinsky replied:
You added all of them by creating pull requests: each pull request is a separate branch.
Once you create a pull request, a corresponding branch is created automatically.
You can then add to the existing pull request by editing/adding/deleting files on the corresponding branch.
That is the preferred way to address pull request comments.
I will accept the `.css` work in a separate pull request this time.
I will also prune the defunct branches this time :)

dubinsky commented on index.html:
```diff
@@ -1,12 +1,70 @@
+	<a href = https://www.google.com>What is special about MW games?</a>
+	</div>
+	</p>
+    <p class = footer>
```
I think HTML has a `<footer>` element now...

CriticalWasTaken replied:
I'll look into it.

dubinsky replied:
see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer

dubinsky commented on index.html:
```diff
@@ -1,12 +1,70 @@
+	</div>
+	</p>
+    <p class = footer>
+		Site is being made by <a href = https://www.google.com>Eliezer Markov.</a>
```
The link should point to something more specific than the Google front page :)

CriticalWasTaken replied:
That is just a placeholder as I'm actually not sure where Mrs. Dubinsky wants this to go.

dubinsky replied:
To something of your choosing I presume ;)

CriticalWasTaken committed:
add <link> to separate css and html.

Assuming this actually finds the correct css file, this should separate the html code from the css code, while leaving the site as it was before.
